client/src/components/Collections/common/modal.js

```diff
@@ -16,13 +16,15 @@ export function collectionCreatorModalSetup(options, Galaxy = null) {
         };
     });
     const showEl = function (el) {
+        const close_event = options.closing_events === undefined || options.closing_events;
         modal.show({
             title: options.title || _l("Create a collection"),
             body: el,
             width: "85%",
             height: "100%",
             xlarge: true,
-            closing_events: true,
```

Comment:
Maybe we should just remove this line here. This would disable closing events and then clicking on the background, or pressing `esc` will not close the modal anymore.

Reply:
That was my first iteration on it (for the RuleBuilder only), just disallowing clicking off the modal as a way to exit the modal. If I disable it here, it'll do it for anything that extends the collectionCreatorModal. I'm fine with that. If I do that though, and some folks are used to just clicking outside of the modal, that will pose a problem.